### DISCUSSION TITLE
Question re: zones

### DISCUSSION THREAD

**Discussion by scottjjr** [blakeblackshear/frigate#5168] [Ask A Question]
Hi, have a question re: zones, and it's not clear whether this is how it works/possible.

I want to have the camera recording as it is now, capturing all people, animals, vehicles, etc coming up and going across my shared driveway.

I would also like a zone on my property that sends an event when a human (i.e. not cars, animals etc) enters the zone.  I will use this to trigger outside lights at night.

I would like the event above to not change what frigate captures/events on within that zone, but be a new event/binary sensor in HA.

Is this possible?

**Comment by NickM-27** [comment]
Yes this is possible. Create the zone and simply don't set `required_zones`. Required zones is something you set and not enabled by default

**Comment by scottjjr** [comment]
Thanks @NickM-27, how does this present in HomeAssistant? I'm wanting to set an automation in HA for when a person or car enters the zone, but all I can see in HA occupancy for object type with no concept of zone.

**Comment by scottjjr** [comment]
Ah the other zones have turned up now, will have a play and see how it goes. Thanks again!
